pages/dice-game/index.js

```diff
@@ -0,0 +1,115 @@
+import styled from "styled-components";
+import GameCard from "@/components/GameCard";
+import { useState } from "react";
+import useSWR from "swr";
+import { SCENARIOS } from "@/utils/scenarios";
+
+export default function DiceGamePage({ isFavorite, toggleFavorite }) {
+  const { data } = useSWR("/api/games");
+  const [currentStory, setCurrentStory] = useState(null);
+  const [suggestedGame, setSuggestedGame] = useState(null);
+
+  const [d20, setd20] = useState(null); //d20 is a conventional way to name 20 sided dice
+
+  const today = new Date().getDay();
+  const intro = SCENARIOS[today].intro; //pick different story every day
+
+  function rollDice() {
+    setd20((prevD20) => {
+      const newD20 = Math.floor(Math.random() * 20) + 1; //choose 20 possible dice rolls to 7 different outcome
+
+      const d20result =
+        newD20 === 1
+          ? "unworthy"
+          : newD20 >= 2 && newD20 <= 4
+          ? "criticalFail"
+          : newD20 >= 5 && newD20 <= 8
+          ? "fail"
+          : newD20 >= 9 && newD20 <= 12
+          ? "neutral"
+          : newD20 >= 13 && newD20 <= 16
+          ? "success"
+          : newD20 >= 17 && newD20 <= 19
+          ? "criticalSuccess"
+          : newD20 === 20 && "godlike";
+
+      const outcomeTitle =
+        d20result.toUpperCase().replace("CRITICAL", "CRITICAL ") + "!";
+      const outcomeText = SCENARIOS[today].outcome[d20result]; //choose 1 outcome with title and text according to dice roll
```

Comment:
why not store the title with the text in the outcome objects? 

```js
outcome: {
  fail: {
    titile: "FAIL!",
    text: "..."
}
}
```
no need for these replacement shenanigans.